packages/circle-react-elements/src/components/TransactionStatusText/TransactionStateText.tsx

```diff
@@ -0,0 +1,71 @@
+import { useEffect } from 'react';
+
+import { TransactionState } from '../../lib/constants';
+import { Badge } from '../ui/badge';
+
+export interface TransactionStateTextProps {
+  /** The state of the transaction **/
+  state:
+    | 'CANCELLED'
```

Comment:
in general I like it more this way, that props are obvious, even if not offered by the Circle SDK. 
But in this case they are available as TransactionState, could you check?

```
import type { TransactionState } from '@circle-fin/developer-controlled-wallets';
```

Reply:
fixed